### ISSUE TITLE
[eztv] Error in logs

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#4412]
I search "vice" in eztv. There are results but if I look in the logs I see this error.

```
CardigannIndexer (eztv): Error while parsing row '<tr name="hover" class="forum_header_border"> 		    <td width="35" class="forum_thread_post" align="center">
		<a href="/shows/823/vice/" title="Vice Torrent">
			<img src="/images/eztv_show_info3.png" border="0" alt="Info" title="Vice Torrents">			
		</a>
	</td> 			<td class="forum_thread_post">                 				<a href="/ep/65349/vice-s03e05-hdtv-x264-batv/" title="VICE S03E05 HDTV x264-BATV [eztv] (179.80 MB)" alt="VICE S03E05 HDTV x264-BATV [eztv] (179.80 MB)" class="epinfo">VICE S03E05 HDTV x264-BATV [eztv]</a> 			</td> 		    <td align="center" class="forum_thread_post">                                               </td>             <td align="center" class="forum_thread_post">179.80 MB</td> 			<td align="center" class="forum_thread_post">3 years</td> 			<td align="center" class="forum_thread_post">-</td> 			<td align="center" class="forum_thread_post_end">
		<a href="/forum/discuss/65349/" rel="nofollow" title="Discuss about VICE S03E05 HDTV x264-BATV [eztv]:">
			<img src="/ezimg/s/1/3/chat_empty.png" border="0" width="16" height="16" alt="Discuss" title="Discuss about this show">			
		</a>
	</td> 		</tr>':

System.Exception: Error while parsing field=download, selector=td:nth-child(3) a.magnet, td:nth-child(3) a, value=<null>: Selector "td:nth-child(3) a.magnet, td:nth-child(3) a" didn't match <tr name="hover" class="forum_header_border"> 		    <td width="35" class="forum_thread_post" align="center">
		<a href="/shows/823/vice/" title="Vice Torrent">
			<img src="/images/eztv_show_info3.png" border="0" alt="Info" title="Vice Torrents">			
		</a>
	</td> 			<td class="forum_thread_post">                 				<a href="/ep/65349/vice-s03e05-hdtv-x264-batv/" title="VICE S03E05 HDTV x264-BATV [eztv] (179.80 MB)" alt="VICE S03E05 HDTV x264-BATV [eztv] (179.80 MB)" class="epinfo">VICE S03E05 HDTV x264-BATV [eztv]</a> 			</td> 		    <td align="center" class="forum_thread_post">                                               </td>             <td align="center" class="forum_thread_post">179.80 MB</td> 			<td align="center" class="forum_thread_post">3 years</td> 			<td align="center" class="forum_thread_post">-</td> 			<td align="center" class="forum_thread_post_end">
		<a href="/forum/discuss/65349/" rel="nofollow" title="Discuss about VICE S03E05 HDTV x264-BATV [eztv]:">
			<img src="/ezimg/s/1/3/chat_empty.png" border="0" width="16" height="16" alt="Discuss" title="Discuss about this show">			
		</a>
	</td> 		</tr>
  at Jackett.Common.Indexers.CardigannIndexer.PerformQuery (Jackett.Common.Models.TorznabQuery query) [0x017a0] in C:\projects\jackett\src\Jackett.Common\Indexers\CardigannIndexer.cs:1480 
```

**Jackett version**:  0.10.589.0
**Mono version**  5.18.0.225

**Comment by karpana** [comment]
I noticed this started happening on my installation at about 7:27pm GMT
My installation is Jackett 0.10.589.0 on Centos7.  I'm not in front of my system to be able to provide the version of Mono, but it was freshly installed last week, so likely the most current.

`Jackett.Common.IndexerException: Exception (eztv): Request to https://eztv.io/ failed (Error BadGateway) - The tracker seems to be down. ---> System.Exception: Request to https://eztv.io/ failed (Error BadGateway) - The tracker seems to be down.
  at Jackett.Common.Indexers.BaseWebIndexer.CheckTrackerDown (Jackett.Common.Utils.Clients.WebClientStringResult response) [0x000ea] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:581 
  at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies (System.String url, System.String cookieOverride, System.String referer, System.Collections.Generic.Dictionary`2[TKey,TValue] headers) [0x000d4] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:446 
  at Jackett.Common.Indexers.CardigannIndexer.PerformQuery (Jackett.Common.Models.TorznabQuery query) [0x00814] in C:\projects\jackett\src\Jackett.Common\Indexers\CardigannIndexer.cs:1229 
  at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x00047] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:331 
   --- End of inner exception stack trace ---
  at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x000e6] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:351 
  at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x00024] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:815 
  at Jackett.Server.Controllers.ResultsController.Torznab (Jackett.Common.Models.DTO.TorznabRequest request) [0x002eb] in C:\projects\jackett\src\Jackett.Server\Controllers\ResultsController.cs:367 `

And a specific manual test within Jackett gives this
`Jackett.Common.IndexerException: Exception (eztv): Request to https://eztv.io/ failed (Error BadGateway) - The tracker seems to be down. ---> System.Exception: Request to https://eztv.io/ failed (Error BadGateway) - The tracker seems to be down.
  at Jackett.Common.Indexers.BaseWebIndexer.CheckTrackerDown (Jackett.Common.Utils.Clients.WebClientStringResult response) [0x000ea] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:581 
  at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies (System.String url, System.String cookieOverride, System.String referer, System.Collections.Generic.Dictionary`2[TKey,TValue] headers) [0x000d4] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:446 
  at Jackett.Common.Indexers.CardigannIndexer.PerformQuery (Jackett.Common.Models.TorznabQuery query) [0x00814] in C:\projects\jackett\src\Jackett.Common\Indexers\CardigannIndexer.cs:1229 
  at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x00047] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:331 
   --- End of inner exception stack trace ---
  at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x000e6] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:351 
  at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x00024] in C:\projects\jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:815 
  at Jackett.Common.Services.IndexerManagerService.TestIndexer (System.String name) [0x00059] in C:\projects\jackett\src\Jackett.Common\Services\IndexerManagerService.cs:222 
  at Jackett.Server.Controllers.IndexerApiController.Test () [0x000e1] in C:\projects\jackett\src\Jackett.Server\Controllers\IndexerApiController.cs:141 
  at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor+TaskOfIActionResultExecutor.Execute (Microsoft.AspNetCore.Mvc.Infrastructure.IActionResultTypeMapper mapper, Microsoft.Extensions.Internal.ObjectMethodExecutor executor, System.Object controller, System.Object[] arguments) [0x00071] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync () [0x00131] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync () [0x0009e] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow (Microsoft.AspNetCore.Mvc.Filters.ActionExecutedContext context) [0x0001b] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next (Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted) [0x00382] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync () [0x0002f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter () [0x0009f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow (Microsoft.AspNetCore.Mvc.Filters.ResourceExecutedContext context) [0x0001b] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next (Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted) [0x00840] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync () [0x0002f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync () [0x0012e] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext httpContext) [0x001cb] in <6092a16d93814eba828b517a2b132f80>:0 
  at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext context) [0x00384] in <427697fe42b7459ba5302fb76d339d3b>:0 
  at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext context) [0x0043e] in <f352e566abf6421e87eafbcf57a0b237>:0 
  at Jackett.Server.Middleware.CustomExceptionHandler.Invoke (Microsoft.AspNetCore.Http.HttpContext httpContext) [0x00042] in C:\projects\jackett\src\Jackett.Server\Middleware\CustomExceptionHandler.cs:29 `

**Comment by kaso17** [comment]
the initial error was caused by some torrent's which don't have any download link. Fixed via b64b3504ba69c4a759c14af6ba95726544005843

The BadGateway error indicates a temporary problem with eztv. It's working fine for me again.
